### ISSUE TITLE
make privacy mitigations normative

### ISSUE THREAD

**Issue by samuelweiler** [w3c/battery#27]
Please either mark section 4 as normative (remove the non-normative mark) or move the mitigations from section 4 into normative parts of the document.

**Comment by anssiko** [comment]
Thanks @samuelweiler. Given the RFC 2119 terms used in this section are SHOULD and MAY, current implementations _should_ remain conformant if we'd remove the non-normative mark. PTAL https://github.com/w3c/battery/pull/43

**Comment by samuelweiler** [comment]
Closing this on the assumption that you'll merge #43

**Comment by anssiko** [comment]
I merged #43. We'll also discuss this change at TPAC 2021 https://github.com/w3c/devicesensors-wg/issues/47 thus I tagged you in the planning issue (timing TBD). Feel free to propose other privacy and security folks who you think we should talk to.

As you know, we are working on revising the security and privacy aspects of the group's specs before we request new horizontal reviews. Having a discussion on the current expectations of these reviews would be helpful. We've scheduled a full session focused on "Privacy, security and architecture" for TPAC.
